multi/imxrt-multi/posixsrv.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Phoenix-RTOS
+ *
+ * i.MX RT builtin posix server (libposixsrv)
+ *
+ * Copyright 2023-2024 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <board_config.h>
+#include "config.h"
+
+#if BUILTIN_POSIXSRV
+
+#include <posixsrv.h>
+#include <sys/threads.h>
+
+
+struct {
+	char stacks[4][0x1000] __attribute__((aligned(8)));
```

Comment:
Do you really need that many threads? It's ~6 KB of RAM each

Reply:
I'm not sure whether to reduce the number of threads - maybe yes, for now I have reduced each thread stack to PAGE_SIZE, but I need to check both targets (on 1064 it worked). Until then, I'm marking this pull-request as WIP.

Reply:
Ok. Looks like works well, tested both rt1064 and rt1176, but need to change `posixsrv_common` to `libposixsrv_common` in multidrv due to name conflict with posixsrv, name seems reasonable.

Reply:
Still, do we need 2 additional `posixsrv_threadMain` threads? It's quite a lot of ram

Reply:
Works when reduced service threads to 1 additional thread:
1 posixsrv_threadRqTimeout
1 posixsrv_threadMain (event: poll/select)
1 (2 previously) additional posixsrv_threadMain (posix services)

![2024-01-26-091520_1369x651_scrot](https://github.com/phoenix-rtos/phoenix-rtos-devices/assets/141153/9cb96122-24f3-4c9d-97e1-0060b66e461f)